nixos/default.nix

```diff
@@ -5,9 +5,20 @@
   pkgs,
   ...
 }:
-with lib;
 
 let
+  inherit (lib)
+    types
+    mkOption
+    mkEnableOption
+    mkIf
+    mkMerge
+    toInt
+    last
+    head
+    escapeShellArg
+    ;
```

Comment:
I hate nixfmt for this ugly ass rat's tail

Reply:
get formatted idiot

Reply:
`nixfmt-classic` (v0.6.0) does not suck like this.